core/src/main/scala/org/apache/spark/storage/BlockManager.scala

```diff
@@ -1231,6 +1231,7 @@ private[spark] class BlockManager(
   def replicateBlock(
       blockId: BlockId,
       existingReplicas: Set[BlockManagerId],
+      excluding: Set[BlockManagerId],
```

Comment:
`blacklist` might be a better name, but this is fine too.

Reply:
Left it as excluding.